f8a_worker/workers/graph_importer.py

```diff
@@ -31,11 +33,14 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('name'))
         self._strict_assert(arguments.get('document_id'))
 
+        rdb = StoragePool.get_connected_storage('BayesianPostgres')
+        ecosystem = Ecosystem.by_name(rdb.session, arguments.get('ecosystem')).name
```

Comment:
If `arguments['ecosystem']` is `redhat-maven`, what will be the value stored in `ecosystem` variable? :)